src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/CommonResources.resx

```diff
@@ -117,12 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BreakingChangesFound" xml:space="preserve">
+    <value>API breaking changes found. If those are intentional, the APICompat suppression file can be updated by running dotnet build /p:GenerateCompatibilitySuppressions=true.</value>
```

Comment:
The MSBuild property was just recently renamed to `ApiCompatGenerateSuppressionFile`: https://github.com/dotnet/sdk/blob/39fcce04a117339a8087060201b1f7797b6167e8/src/ApiCompat/Microsoft.DotNet.ApiCompat.Task/build/Microsoft.DotNet.ApiCompat.ValidateAssemblies.Common.targets#L20

The old property still works for compat reasons but let's use the new one going forward.

Reply:
```suggestion
    <value>API breaking changes found. If those are intentional, the APICompat suppression file can be updated by running "dotnet build /p:ApiCompatGenerateSuppressionFile=true".</value>
```

Reply:
This command only applies to the MSBuild task frontend. The CLI requires a different command. Can you please add something like "or --generate-suppression-file"? I think you already had that before.